illuminate.py

```diff
@@ -1,22 +1,24 @@
+import hydra
 import omegaconf
```

Comment:
I think this line can be removed now (together with the omegaconf type hints further below).

Reply:
Done